artifacts_test.py

```diff
@@ -16,18 +16,68 @@
 
 from sdcm.cluster import SCYLLA_YAML_PATH
 from sdcm.tester import ClusterTester
-
+from sdcm.utils.housekeeping import HousekeepingDB
 
 STRESS_CMD: str = "/usr/bin/cassandra-stress"
 
 
 class ArtifactsTest(ClusterTester):
+
+    REPO_TABLE = "housekeeping.repo"
+    CHECK_VERSION_TABLE = "housekeeping.checkversion"
+
+    # TODO: uncomment setUp and tearDown when problem with SSL connection to MYSQL will be solved
+    # def setUp(self) -> None:
+    #     super().setUp()
+    #
+    #     self.housekeeping = HousekeepingDB.from_keystore_creds()
+    #     self.housekeeping.connect()
+    #
+    # def tearDown(self) -> None:
+    #     self.housekeeping.close()
+    #
+    #     super().tearDown()
+
+    # TODO: uncomment check_version when problem with SSL connection to MYSQL will be solved
+    # def check_version(self, prev_id: int, new_row_expected: bool) -> int:
```

Comment:
name is misleading. Which version? Where it checks? What is the check? find better name

Reply:
changed to `check_scylla_version_in_housekeepingdb`